psi4/share/psi4/fsapt/fsaptdiff.py

```diff
@@ -57,62 +57,62 @@
 
     fh = open('%s/fsapt.dat' % dirA, 'w')
     fh, sys.stdout = sys.stdout, fh
-    print '  ==> F-ISAPT: Links by Charge <==\n'
+    print('  ==> F-ISAPT: Links by Charge <==\n')
     stuffA = computeFsapt(dirA, False)
-    print '   => Full Analysis <=\n'
+    print('   => Full Analysis <=\n')
     printOrder2(stuffA['order2'], stuffA['fragkeys']) 
-    print '   => Reduced Analysis <=\n'
+    print('   => Reduced Analysis <=\n')
     printOrder2(stuffA['order2r'], stuffA['fragkeysr']) 
```

Comment:
Hmm, no, no restriction on Py version (there was at one time, though). Looks py pb11 is suggesting only py3 and we're sneaking py2 in there by setting -DPY every time. (This is just me looking at the CMake. In my opinion, the only real test of CMake behavior is to run it under the different conditions.) We may be able to offload py detection entirely to pb11 now.